src/components/reactTasks/comment.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+
+export default class Comment extends React.Component{
+
+  constructor(){
+    super()
+    this.state = {comments: ['Brilliant!', 'Terrible']}
+  }
+
+
+ handleChange(event) {
+    this.setState({comments: event.target.comments});
```

Comment:
This is setting your array of comments to the string that is being typed in the input, you need another value in state `comment` for storing the comment that is being typed.